server/@types/models/Referral.ts

```diff
@@ -6,6 +6,7 @@ type Referral = {
   oasysConfirmed: boolean
   offeringId: CourseOffering['id']
   prisonNumber: Person['prisonNumber']
+  reason: string
```

Comment:
Can we update `ReferralUtils.isReadyForSubmission` and relevant tests with this change?